package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "JS-To-Do-List",
+  "version": "1.0.0",
+  "description": "Learning the concepts of Single Responsibility and Tightly Coupled Objects in Javascript",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Great job implementing a script to use Webpack. However, it's best practice to create at least 2 scripts for your application, one for development and another one for production. We need to also to inform in which [mode](https://webpack.js.org/configuration/mode/) Webpack will run in these scripts. We will have something like this:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
  },
```

- [ ] We also need to inform users about these script in the README file. This information can be placed in the Usage section. This way users will know how they can use your application.

**OPTIONAL**

- [ ] You can create an even better script for development stages. You will need to add the `
webpack-dev-server` and create a script like this:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
Running the `start` script will automatically open the index file in the browser, without the need to open it manually